packages/experiments-realm/app-task-card.gts

```diff
@@ -349,59 +353,78 @@ class AppTaskCardIsolated extends Component<typeof AppCard> {
     return this.selectedItems.get(filterType) ?? [];
   }
 
+  @action getFilterIcon(filterType: FilterType) {
+    switch (filterType) {
```

Comment:
there is already a static configuration for all the filters. Let's keep all data in one place